shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -496,6 +497,10 @@ private Drawable getSplashScreenFromManifest() {
     } catch (PackageManager.NameNotFoundException e) {
       // This is never expected to happen.
       return null;
+    } catch (Resources.NotFoundException e) {
```

Comment:
I still think it's appropriate to propagate the `Resources.NotFoundException` instead of catching it here if the app specifies a resource ID that does not exist.

IIUC https://github.com/flutter/flutter/issues/42663 involved an exception happening when the app did not provide a value for `SPLASH_SCREEN_META_DATA_KEY`.  That exception should already be fixed in the current master.

If the app does provide a value and that value is invalid, then I'd prefer to throw instead of warning.